[1.9.6 regression] 'find_package(jsoncpp REQUIRED)' fails: Unknown CMake command "check_required_components"

**Describe the bug**
```
$ cat CMakeLists.txt 
find_package(jsoncpp REQUIRED)
```

```
CMake Error at /usr/local/lib/cmake/jsoncpp/jsoncppConfig.cmake:31 (check_required_components):
  Unknown CMake command "check_required_components".
Call Stack (most recent call first):
  CMakeLists.txt:1 (find_package)
```

**Desktop (please complete the following information):**
 - OS: FreeBSD 14.1
 - cmake-3.29.6
 - Meson version: 1.5.1

--- a/jsoncppConfig.cmake.meson.in
+++ b/jsoncppConfig.cmake.meson.in
@@ -4,5 +4,3 @@
 @MESON_STATIC_TARGET@
 
 include ( "${CMAKE_CURRENT_LIST_DIR}/jsoncpp-namespaced-targets.cmake" )
-
-check_required_components(JsonCpp)

